apache-tapestry/src/main/resources/com/baeldung/tapestry/components/Layout.tml

```diff
@@ -0,0 +1,21 @@
+<html xmlns:t="http://tapestry.apache.org/schema/tapestry_5_4.xsd">
+	<head>
+	    <title>${title}</title>
```

Comment:
seems to be using a combination of spaces and tabs to indent; let's stick to spaces

Reply:
fixed the indentation